app/decorators/physical_server_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class PhysicalServerDecorator < MiqDecorator
+  def fileicon
+    "svg/vendor-#{vendor.try(:downcase)}.svg"
```

Comment:
@walteraa @himdel @skateman I've just created a PR([#15129](https://github.com/ManageIQ/manageiq/pull/15129)) to fix the factory.